apps/admin-gui/src/app/shared/components/dialogs/add-member-to-resource/add-member-to-resource.component.html

```diff
@@ -0,0 +1,93 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.ADD_MEMBER_TO_RESOURCE.TITLE' | translate}}</h1>
+  <mat-horizontal-stepper #stepper [linear]="true">
+    <div class="dialog-container" mat-dialog-content>
+      <mat-step completed=false>
+        <ng-template matStepLabel>Select resource</ng-template>
+        <div>
+          <span class="font-weight-bold">{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.FILTER' | translate}}</span>
+          <div>
+          <mat-form-field class="input-style">
+            <input [formControl]="facilityCtrl" [matAutocomplete]="autoFacility"
+                   aria-label="Number"
+                   matInput
+                   type="text">
+            <mat-autocomplete #autoFacility="matAutocomplete" autoActiveFirstOption>
+              <mat-option *ngFor="let facility of filteredFacilities | async" [value]="facility">
+                {{facility}}
+              </mat-option>
+            </mat-autocomplete>
+          </mat-form-field>
+          </div>
+        </div>
+        <div>
+          <span class="font-weight-bold">{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.SELECTED' | translate}}</span>
+          <div>
+          <mat-form-field class="input-style">
+            <mat-select disableOptionCentering="true">
+              <mat-option *ngFor="let resource of filteredResources | async" [value]="resource"
+                          (click)="setResource(resource);
+                           stepper.selected.completed = true">
+                {{resource.name}}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+          </div>
+        </div>
+        <div>
+          <span class="font-weight-bold">{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.DESCRIPTION' | translate}}</span>
+          <div class="ml-5"> {{description}} </div>
+        </div>
+        <div>
+          <span class="font-weight-bold">{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.SERVICES' | translate}}</span>
+          <div class="ml-5" *ngFor="let service of services">{{service.name}}</div>
+        </div>
+
+        <div mat-dialog-actions>
+          <button
+            (click)="onCancel()"
+            class="ml-auto"
+            mat-flat-button>{{'DIALOGS.ADD_MEMBER_TO_RESOURCE.CANCEL' | translate}}</button>
+          <button
+            [disabled]="selectedResource == null"
+            (click)="loadGroups()"
+            class="ml-2"
+            color="accent"
+            mat-flat-button
+            matStepperNext
+          >{{'DIALOGS.ADD_MEMBER_TO_RESOURCE.CONTINUE' | translate}}</button>
+        </div>
+      </mat-step>
+      <mat-step>
+        <ng-template matStepLabel>Add group</ng-template>
```

Comment:
please translate this and also I think the better label for this could be "Select group"